precompiles/staking/types.go

```diff
@@ -22,6 +22,11 @@ import (
 	cmn "github.com/evmos/evmos/v16/precompiles/common"
 )
 
+const (
+	DoNotModifyCommissionRate    = -1 // constant used in flags to indicate that commission rate field should not be updated
+	DoNotModifyMinSelfDelegation = -1 // constant used in flags to indicate that min self delegation field should not be updated
```

Comment:
Nit: coherence with other comments

```suggestion
        // Constant used in flags to indicate that commission rate field should not be updated.
	DoNotModifyCommissionRate    = -1
	// Constant used in flags to indicate that min self delegation field should not be updated.
	DoNotModifyMinSelfDelegation = -1 
```